modules/extensions-spring-support/src/test/resources/models/semantic-terms-extension.json

```diff
@@ -3453,40 +3453,6 @@
                   "name": "endpoint",
                   "description": "",
                   "modelProperties": {}
-                },
-                {
-                  "type": {
-                    "format": "java",
-                    "type": "@ref:org.mule.sdk.api.annotation.semantics.connectivity.NtlmDomain"
-                  },
-                  "hasDynamicType": false,
-                  "required": true,
-                  "isConfigOverride": false,
-                  "isComponentId": false,
-                  "fieldValueProviderModels": [],
-                  "expressionSupport": "SUPPORTED",
-                  "role": "BEHAVIOUR",
-                  "dslConfiguration": {
-                    "allowsInlineDefinition": false,
-                    "allowsReferences": true,
-                    "allowTopLevelDefinition": false
-                  },
-                  "layoutModel": {
-                    "password": false,
-                    "text": false,
-                    "query": false,
-                    "order": 8
-                  },
-                  "allowedStereotypeModels": [
-                    {
-                      "type": "NTLM_DOMAIN",
-                      "namespace": "SEMANTIC-TERMS"
-                    }
-                  ],
-                  "semanticTerms": [],
-                  "name": "ntlmDomain",
```

Comment:
Does anyone know why this parameter was added in the model? I see no reference to the class it is in.